compiler/rustc_codegen_llvm/src/builder.rs

```diff
@@ -606,6 +610,16 @@ impl<'a, 'll, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'll, 'tcx> {
         }
     }
 
+    fn noundef_metadata(&mut self, load: &'ll Value) {
```

Comment:
As this is only called from cg_llvm code and a pretty LLVM-specific concept, maybe move this to `impl Builder` rather than including it in the `BuilderMethods` trait?